packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf

```diff
@@ -430,14 +430,15 @@ dbms.connector.https.enabled=true
 # The map should be formatted as a semicolon separated list of key-value pairs, where the
 # key is the LDAP group name and the value is a comma separated list of corresponding role names.
 # E.g. group1=role1;group2=role2;group3=role3,role4,role5
-#dbms.security.ldap.authorization.group_to_role_mapping=
-
+#
```

Comment:
I think according to the policy this should be here to show that the default value is empty